src/screens/ProfileScreen/index.js

```diff
@@ -635,12 +628,12 @@ const ProfileScreen = ({route}) => {
         style={styles.flatlistContainer}
         onScroll={handleScroll}
         ListFooterComponent={isFetchingList ? <ActivityIndicator /> : null}
-        onEndReach={handleOnEndReached}
         initialNumToRender={2}
         maxToRenderPerBatch={2}
         updateCellsBatchingPeriod={10}
         removeClippedSubviews
         windowSize={10}
+        onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been removed and replaced with `onMomentumScrollEnd`. This change seems to be part of the optimization effort to preload posts before the user reaches the end of the feed. Just ensure that `fetchNextFeeds` function handles all edge cases properly, like when there are no more posts to fetch or if an error occurs during fetching.


<!-- This is an auto-generated comment by OSS CodeRabbit -->